mkdocs.yml

```diff
@@ -0,0 +1,87 @@
+# Configuration for all pages
+site_name: "MCAP"
+theme:
+  name: material
+  favicon: img/favicon32.png
+  logo: img/mcap.png
+  palette:
+    - scheme: default
+      toggle:
+        icon: material/brightness-7
+        name: Switch to dark mode
+      primary: "white"
+    - scheme: slate
+      toggle:
+        icon: material/brightness-4
+        name: Switch to light mode
+      primary: "black"
+  font:
+    text: Open Sans
+markdown_extensions:
```

Comment:
```suggestion
extra_css: extra.css

markdown_extensions:
```

Suggest adding this CSS to fix the cut-off github info:

<img width="246" alt="image" src="https://user-images.githubusercontent.com/14237/183757454-40ec7877-cf8e-4a46-b5fa-cac410b5f446.png">

```css
.md-source__fact--version {
  display: none;
}
.md-source__fact--stars::before {
  margin-left: 0;
}
.md-source__fact::before {
  vertical-align: bottom;
}
```